Jenkinsfile

```diff
@@ -101,6 +107,16 @@ pipeline {
     }
 
     stages {
+        stage('Install Node.js 10') {
+            steps {
+                timeout(time: 10, unit: 'MINUTES') {
+                    // We need Node.js 10 or later
+                    sh "curl -fsSL https://deb.nodesource.com/setup_10.x | sudo -E bash -"
+                    sh "sudo apt-get install -y nodejs"

```

Comment:
@MarkAckert Could the agent be updated to use a newer Node.js version? Running `typedoc` in the Jenkins pipeline requires Node 10 or later. It might make sense to upgrade from 8 straight to 12, since 10 is EOL in a week.

Reply:
I updated the agent with Node 12. Let me know if you need anything else changed on the build img